.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Create Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - '[0-9].[0-9]+.[0-9]+' # Push events to any matching semantic tag. For example, 1.10.1 or 2.0.0.
+        # For more details, see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+        # and https://docs.npmjs.com/about-semantic-versioning
+
+jobs:
+  build:
+    runs-on: ${{matrix.os}}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Test
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install
+      - run: cp env.example .env
+      - run: npm run test
+  Prettier-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: npm install && npm run prettier-check
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: npm install && npm run eslint
+  release:
+    needs: [build, Prettier-Check, Lint]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+        # Update version in package.json according to the tag
+      - name: Check package version
+        uses: technote-space/package-version-check-action@v1
```

Comment:
I think we should have people do this with `npm version` locally, so they can update package.json and also tag before they push.  The the tag will match the commit that bumps the package version (vs. adding a new commit afterward).

Reply:
You're right, the package.json update commit would now be the next after the tagged one:
![image](https://user-images.githubusercontent.com/33902374/106540478-054e7300-64ce-11eb-8a54-2838cf7e86c4.png)
I will try to see if we can do anything about it with GitHub Actions and switch to `npm release` in case we cannot.

Reply:
Could not find a way to match the tagged commit with the one that updates the version in `package.json` with GitHub Actions. Will get rid of `technote-space/package-version-check-action@v1` and switch to use `npm version` locally instead.